lib/laa_crime_forms_common/pricing/nsm/calculators/totals.rb

```diff
@@ -40,14 +40,18 @@ def cost_summary
             @cost_summary ||= Calculators::CostSummary.call(claim, work_types, letters_and_calls, rates:)
           end
 
+          def cost_summary_for_totals
```

Comment:
So really you want to include an `additional_fees` section in the Totals payload, because that's where we put breakdowns of things that get included in the cost summary. `additional_fees` is similar to `letters_and_calls` in that the value gets added to profit costs in the cost summary, but we also want a breakdown available in order to populate a standalone table in the UI.